os/arch/arm/src/armv8-m/up_assert.c

```diff
@@ -100,13 +100,17 @@
 #include <sys/boardctl.h>
 #endif
 #ifdef CONFIG_BINMGR_RECOVERY
-#include <unistd.h>
 #include <stdbool.h>
+#include <unistd.h>
 #include <queue.h>
 #include <tinyara/wdog.h>
 #include "semaphore/semaphore.h"
 #include "binary_manager/binary_manager.h"
 #endif
+#if defined(CONFIG_DEBUG_WORKQUEUE) && defined(__KERNEL__)
+#include <string.h>
```

Comment:
Here is no string function. It's in wqueue.h. Let's move this including into the header.